docs/gl_objects/merge_request_approvals.rst

```diff
@@ -58,22 +58,9 @@ Change project-level or MR-level MR approvals settings::
     p_mras.approvals_before_merge = 2
     p_mras.save()
 
-    mr.approvals.set_approvers(approvals_required=1)
-    # or
```

Comment:
The MR-level `set_approvers()` is still valid (although maybe could use a refactor too). We can leave it here.

Reply:
Duh See what happens when I work on this stuff late at night. I thought there was only one `set_approvers()` method. But now I clearly see that one still exists.